README.md

```diff
@@ -1,66 +1,144 @@
+## Quickstart
 
-1. To test your changes in the app run,
-   `yarn start`
-   The app tries to process all ember & ember-data versions since 1.0 which takes high memory & time to complete. If you intend it, then run `yarn start --max_old_space_size=8192`.
-   You are setting your node max heap space to 8GB, so make sure you have that much space available on your machine.
+### Generate Docs from Ember Source
 
-## To Generate docs for a specific project and/or version for development
+You can generate docs from your local copies of [emberjs/ember.js](https://github.com/emberjs/ember.js/) and [emberjs/data](https://github.com/emberjs/data) repositories and serve them locally as [JSON:API](http://jsonapi.org/).
 
-You can do this by passing `--project ember/ember-data --version 2.11.1` as an argument to the index script. e.g., `yarn start --project ember --version 2.11.0`.
-You need an additional flag `AWS_SHOULD_PUBLISH=true` for publishing the docs.
+```bash
+# Clone Ember.js and Ember Data repositories.
+git clone https://github.com/emberjs/ember.js/
+git clone https://github.com/emberjs/data
 
-## To override a specific version of a doc with a different yuidoc from your machine (For core contributors)
+# Clone the repository into the same root folder.
+git clone https://github.com/ember-learn/ember-jsonapi-docs
+cd ember-jsonapi-docs
 
-- Read this section first!
-  - We assume you have the keys to the kingdom before you start doing this (AWS keys to publish to api-docs.emberjs.com & all necessary env variables that need to be set) 😄
-  - In its present form this should be used only when there aren't new docs out there that are yet to be processed. As in if ember 3.3 is released but isn't indexed yet you should wait for this app to finish processing it via the cron job we have on heroku before you can proceed to modify any of the existing docs from your machine.
-- First run `yarn start` so that you have all the docs from s3 on your local. This is important so that we don't loose other versions of docs that are already out there when the index files are generated
-- Then go to `./tmp/s3-docs/<the_version_you_want_to_replace>` and override the file there with the yuidoc file that you want to be processed. Ensure that the file name is same as the one that's already there.
-- Then run `yarn start --project=ember-data --version=3.2.0 --ignorePreviouslyIndexedDoc`. Make sure you enter the entire version(including patch version).
-- To run against all versions of ember and ember-data regardless of indexed version, run `node --max_old_space_size=8192 index.js --clean`
+# Install the dependencies.
+yarn
 
-## Generating API Documentation and Testing API Docs Locally
+# Generate docs for a particular project/release.
+yarn gen --project ember --version 3.17.0
+yarn gen --project ember-data --version 3.17.0
```

Comment:
Another gotcha: the version here should match the one in the package.json of your project, but you can't use 'beta' or 'canary'. So for example if the ember-data package.json says "3.19.0-beta.2" use "3.19.0".

Reply:
Got it! Thank you for sharing! I'll create another PR to include this information.